src/styles/book.css

```diff
@@ -0,0 +1,120 @@
+.d-flex {
```

Comment:
- [ ] Let's give the book cards a white background with gray borders. This will help our design to match better with the template given. You can find the background color, border attributes, and border-radius by inspecting the cards on the Zeplin design.

![image](https://user-images.githubusercontent.com/79114233/142738756-0d89ef91-ded6-4e11-aa66-d73eedaa0048.png)

Yours:
![image](https://user-images.githubusercontent.com/79114233/142738763-33626e4f-6b4c-43c2-99d4-a79860a046f4.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/142738771-2c0d1a51-785e-4212-b3c3-c706bcbefbde.png)

Reply:
- [ ] You might want to also change the background of the whole page since only the navbar and the cards have a white background. You can see the actual background color used by inspecting the background in the Zeplin template: `background-color: #fafafa`

![image](https://user-images.githubusercontent.com/79114233/142738844-d41f5145-f9d9-4957-b4e9-e69f5992216b.png)